src/components/radio-group/radio-group.js

```diff
@@ -2,17 +2,21 @@ import React from 'react';
 import { connect } from 'react-redux';
 import { capitalize } from 'lodash';
 import Component from './radio-group-component';
-import actions from 'redux_modules/metadata';
+import metadataActions from 'redux_modules/metadata';
+import * as googleAnalyticsActions from 'actions/google-analytics-actions';
```

Comment:
The same for this, maybe importing the methods we are using instead of all the actions.
It might have some performance and bundle benefits:
https://stackoverflow.com/questions/45305272/is-there-a-performance-difference-between-import-as-from-lodash-and-import